rfcs/formats/network/iin_method.md

```diff
@@ -0,0 +1,264 @@
+<!--
+ Copyright IBM Corp. All Rights Reserved.
+
+ SPDX-License-Identifier: CC-BY-4.0
+ -->
+# IIN DID Method
+
+
+- RFC: 03-021
+- Authors: Bishakh Chandra Ghosh, Venkatraman Ramakrishna, Krishnasuri Narayanam, Ermyas Abebe
+- Status: Proposed
+- Since: 10-March-2022
+
+
+
+## Introduction
+
+An Interoperation Identity Network (IIN), is a system built on a distributed shared ledger that supports registration of DIDs to networks (e.g. permissioned consortium networks) and their units, as well as resolution of network DID documents form their DID. The IIN also has reputed trust anchors, which play the important role of certifying (or validating) the identity of owners of those DIDs, as well as their memberships in their respective networks, in effect creating roots of trust for cross-network interoperations. Thus, the IIN provides the foundation for our architecture to enable blockchain network-identity discovery and validation. The IIN DID Method specification conforms to the requirements specified in [Decentralized Identifiers (DIDs) v1.0](https://w3c.github.io/did-core/). The IIN as a system is detailed further [here](./iin.md).
```

Comment:
", is a system built..." --> no comma
"DIDs to networks....." --> "DIDs representing networks......"
"documents form their......" --> documents from their......"
"An IIN is also associated with one or more reputed trust anchors, which play....."
Also link "trust anchors" to the Indy tutorial where they are described?
"Thus, IINs provide a foundation to enable the discovery and validation of blockchain/DLT networks and their participants." (making the language neutral and not specific to Weaver's needs)